apps/teams-test-app/e2e-test-data/hostEntity.tab.json

```diff
@@ -11,15 +11,20 @@
   },
   "testCases": [
     {
-      "title": "addAndConfigure API Call - Success",
+      "title": "addAndConfigure API Call - Success with meetingParams and appTypes",
       "type": "callResponse",
       "boxSelector": "#box_addAndConfigure",
       "inputValue": {
         "hostEntityIds": {
           "threadId": "threadId"
-        }
+        },
+        "appTypesOrMeetingParams": {
+          "isTownhall": true,
+          "isStreamingThread": true
+        },
+        "appTypes": ["web"]
```

Comment:
Will the appType follow the interface value as edu here? FYI: The appType is an optional value, and we haven't used it for any logic on the TMP side yet.
![image](https://github.com/user-attachments/assets/995bf04a-64e6-48b6-986d-ea96a1e1f446)